content/en/agent/remote_config/_index.md

```diff
@@ -204,10 +198,43 @@ Gain visibility into the Remote Configuration status of your Agent through the [
-  | CONNECTION ERROR        |   The Agent deployed in your environment has `remote_config.enabled` set to true in its `datadog.yaml` configuration file, however, the Agent cannot be found in the Remote Configuration service. The most likely cause is that the Agent is unable to reach [Remote Configuration endpoints][17]. To fix the issue, allow outbound HTTPS access to Remote Configuration endpoints from your environment. This status displays when the Agent version is `7.45.0` or higher. 
+  | CONNECTION ERROR        |   The Agent deployed in your environment has `remote_config.enabled` set to true in its `datadog.yaml` configuration file, however, the Agent cannot be found in the Remote Configuration service. The most likely cause is that the Agent is unable to reach Remote Configuration [endpoints][17]. To fix the issue, allow outbound HTTPS access to Remote Configuration endpoints from your environment. This status displays when the Agent version is `7.45.0` or higher. 
   | DISABLED       |   The Agent deployed in your environment has `remote_config.enabled` set to false in its `datadog.yaml` configuration file. Set `remote_config.enabled` to true if you want to enable Remote Configuration on the Agent. This status displays when the Agent version is `7.45.0` or higher. | 
   | NOT CONNECTED       | The Agent cannot be found in the Remote Configuration service and could have `remote_config.enabled` set to true or false in its `datadog.yaml` configuration file. Check your local Agent configuration or your proxy settings. This status displays when the Agent version is higher than `7.41.1` but lower than `7.45.0`.            | 
   | UNSUPPORTED AGENT   | The Agent is on a version that is not Remote Configuration capable. To fix this issue, update the Agent to the latest available version. |
+
+## Opting out of Remote Configuration at the Agent level
+
+Starting with Agent version 7.47.0, remote_configuration.enabled is set to true by default in the Agent. This will cause the agent to request configuration updates from the Datadog site. To receive configurations from Datadog, you still need to enable Remote Configuration at the organization level and enable Remote Configuration capability on your API Key from the Datadog UI, and allow outbound HTTPS access to Remote Configuration [endpoints][17] from your environment. If you don't want your agent to send configuration requests to Datadog, you can set remote_configuration.enabled to false in the Agent.
+
+{{< tabs >}}
+{{% tab "Configuration YAML file" %}}
+Change remote_configuration.enabled from true to false in your configuration YAML file:
```

Comment:
Can you add any more information about the configuration file? What the filename is, and where it is located, perhaps?

Reply:
The filename is datadog.yaml and location varies. We have a link in our page here:
https://docs.datadoghq.com/agent/guide/agent-configuration-files/?tab=agentv6v7#agent-main-configuration-file
I will link to it.